R/aggregate_alignment_loanbook_exposure.R

```diff
@@ -5,46 +5,127 @@
 #' @param level Character. Vector that indicates if the aggregate alignment
 #'   metric should be returned based on the net technology deviations (`net`) or
 #'   disaggregated into buildout and phaseout technologies (`bo_po`).
+#' @param .by <tidy-select> Character. Optionally, a selection of columns to
+#'   group by, instead. All columns indicated must be available variables in the
+#'   `matched` data set. The intended use case is to allow analyzing the loan
+#'   books by additional traits of interest, such as types of financial
+#'   institutions.
```

Comment:
NIT: Maybe mention that critical columns can't be used as grouping columns? (e.g. you can't group by `loan_size_outstanding`)

Reply:
probably a good idea, but maybe there should be a check that ensures this is not happening then